api/src/db/migrations/versions/2024_05_01_add_created_at_updated_at_and_deleted_.py

```diff
@@ -0,0 +1,1168 @@
+"""Add created_at, updated_at, and deleted_at columns to staging tables
+
+Revision ID: 1ddd1d051a99
+Revises: e3a1be603d26
+Create Date: 2024-05-01 11:14:34.332661
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "1ddd1d051a99"
+down_revision = "e3a1be603d26"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table("topportunity_cfda", schema="legacy")
+    op.drop_table("t1", schema="api")
+    op.drop_table("tfundinstr_synopsis_hist", schema="legacy")
+    op.drop_table("tfundactcat_forecast_hist", schema="legacy")
+    op.drop_table("t2", schema="api")
+    op.drop_table("tfundinstr_synopsis", schema="legacy")
```

Comment:
You've got a lot going on in this migration that isn't the 3 timestamps you added, something is off here.

Reply:
Looks like its dropping the legacy foreign tables - I think we want to modify env.py to ignore anything in that schema for alembic

Reply:
Thanks, fixed